libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -58,7 +59,11 @@ void SRV_Channel::output_ch(void)
                 // non-mapped rc passthrough
                 int16_t radio_in = c->get_radio_in();
                 if (passthrough_mapped) {
+                  if (rc().in_rc_failsafe() && ((1U<<passthrough_from) & SRV_Channels::get_rc_fs_mask())) {
```

Comment:
the if() should be in the other order to be slightly faster

Reply:
could access the fs mask more directly?

Reply:
done using Peters suggestion below